app/models/account_authority_history.rb

```diff
@@ -27,4 +27,14 @@ class AccountAuthorityHistory < ApplicationRecord
 
   serialize :authorized_voters_before, JSON
   serialize :authorized_voters_after, JSON
+
+  API_FIELDS = %i[
+    authorized_voters_after
+    authorized_voters_before
+    authorized_withdrawer_after
+    authorized_withdrawer_before
+    network
+    created_at
+    updated_at
```

Comment:
In this case I think created_at would be enough. But OK to leave it if you think it might be useful in some cases.